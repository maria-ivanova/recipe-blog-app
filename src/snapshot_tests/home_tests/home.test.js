import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import Home from '../../components/home.js';

jest.mock('../../components/indexBanner.js', () => 'IndexBanner');
jest.mock('../../components/itemsList.js', () => 'ItemsList');
jest.mock('../../components/indexInfo.js', () => 'IndexInfo');

describe('Home Component', () => {
    it('should render layout', () => {
        const tree = renderer.create(
            <Routing>
                <Home/>
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})