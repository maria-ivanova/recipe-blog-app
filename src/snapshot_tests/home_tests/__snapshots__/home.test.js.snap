// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component should render layout 1`] = `
Array [
  <IndexBanner />,
  <ItemsList
    maxElements={8}
    sortCriterion="createdDate"
  />,
  <IndexInfo />,
  <ItemsList
    maxElements={8}
    sortCriterion="likes"
  />,
]
`;
